src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs

```diff
@@ -299,102 +292,84 @@ public static partial class RoslynInsertionTool
                         }
                         catch (EmptyCommitException ecx)
                         {
-                            isInsertionCancelled = true;
-
-                            if (latestBuild != null && latestBuild.Result != BuildResult.Succeeded)
-                            {
-                                noProgressOnFailedBuilds = true;
-                            }
-
-                            Log.Warn($"Unable to create pull request for '{branch.FriendlyName}'");
-                            Log.Warn(ecx);
-                            return;
+                            Console.WriteLine($"Unable to create pull request for '{branch.FriendlyName}'");
+                            Console.WriteLine(ecx);
+                            return false;
                         }
                         catch (Exception ex)
                         {
-                            Log.Error($"Unable to create pull request for '{branch.FriendlyName}'");
-                            Log.Error(ex);
-                            return;
+                            Console.WriteLine($"Unable to create pull request for '{branch.FriendlyName}'");
+                            Console.WriteLine(ex);
+                            return false;
                         }
                     }
 
                     if (pullRequest == null)
                     {
-                        Log.Error($"Unable to create pull request for '{branch.FriendlyName}'");
-                        return;
+                        Console.WriteLine($"Unable to create pull request for '{branch.FriendlyName}'");
+                        return false;
                     }
                 }
 
                 // ********************* Create validation build *****************************
                 if (Options.QueueValidationBuild)
                 {
                     cancellationToken.ThrowIfCancellationRequested();
-                    Log.Info($"Create Validation Build");
+                    Console.WriteLine($"Create Validation Build");
 
                     if (pullRequest == null)
                     {
-                        Log.Error("Unable to create a validation build: no pull request.");
-                        return;
+                        Console.WriteLine("Unable to create a validation build: no pull request.");
+                        return false;
                     }
 
                     try
                     {
-                        await QueueBuildPolicy(pullRequest, "CloudBuild - Request RPS");
```

Comment:
This, in particular, will need to be reverted. We should also look through the rest of this to be sure other changes weren't lost.